recipes/repdenovo/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.0.1" %}
+{% set sha256 = "e5d05f73bd204787d9371f8def49280bdafecf35553662e207c1cd171c0fc516" %}
+package:
+  name: repdenovo 
+  version: '{{version}}'
+
+source:
+  git_url: https://github.com/Reedwarbler/REPdenovo.git
+  patches:
+    - TERefiner.Makefile.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python=2.7
+    - zlib
+    - jellyfish
```

Comment:
FYI, jellyfish is an ambiguous package name at the moment. We've started renaming it in bioconda since a different package with the same name exists in conda-forge. Xref: https://github.com/bioconda/bioconda-recipes/pull/17748

Reply:
Thanks for the tip. What should I rename it to?

Reply:
kmer-jellyfish